### PR TITLE
Add lend command

### LINKED ISSUE #17: As a person with a lot of friends I can keep track of who owes me what on which day
... so that I can ask them to pay me back

### LINKED ISSUE #22: As a rich student who always lends people money I can take note of who owes me money
... so that I can track them and ask them for it back.

### LINKED ISSUE #27: As a calculative person I can keep track of how much exactly my friends owe me
... I can get all my money back.

### PR DESCRIPTION
**Pull request by syinyichen**

### CODE REVIEW

File: src/main/java/seedu/address/model/transaction/Loan.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+     */
+    public Loan(Description description, Amount amount, LocalDate date) {
+        super(description, amount, date, new Tag("Loan"));
+    }
+
+    /**
+     * Checks if both Loans are the same.
+     */
+    @Override
+    public boolean equals(Object other) {
```
Comment (CheyanneSim) [inline src/main/java/seedu/address/model/transaction/Loan.java]: I feel like equals, hashcode and toString don't need cus Transaction class already have them.

File: src/main/java/seedu/address/storage/JsonAdaptedPerson.java [hunk 2]
```diff
@@ -38,13 +40,17 @@
...
                              @JsonProperty("debts") List<JsonAdaptedDebt> debts,
+                             @JsonProperty("loans") List<JsonAdaptedLoan> loans,
                              @JsonProperty("tagged") List<JsonAdaptedTag> tagged) {
         this.name = name;
         this.phone = phone;
         this.email = email;
         if (debts != null) {
             this.debts.addAll(debts);
         }
+        if (debts != null) {
```
Comment (CheyanneSim) [inline src/main/java/seedu/address/storage/JsonAdaptedPerson.java]: i think should be if (loans != null) haha

File: src/main/resources/view/PersonListCard.fxml [hunk 3]
```diff
@@ -1,18 +1,10 @@
...
-<?import javafx.scene.control.TableColumn?>
-<?import javafx.scene.control.TableView?>
-<?import javafx.scene.layout.ColumnConstraints?>
-<?import javafx.scene.layout.FlowPane?>
-<?import javafx.scene.layout.GridPane?>
-<?import javafx.scene.layout.HBox?>
-<?import javafx.scene.layout.Region?>
-<?import javafx.scene.layout.RowConstraints?>
-<?import javafx.scene.layout.VBox?>
+<?import javafx.geometry.*?>
```
Comment (CheyanneSim) [inline src/main/resources/view/PersonListCard.fxml]: Question: Are these .* wildcard imports?

File: src/test/java/seedu/address/logic/parser/SharkieParserTest.java [hunk 4]
```diff
@@ -64,52 +66,61 @@ public void parseCommand_edit() throws Exception {
         Person person = new PersonBuilder().build();
         EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(person).build();
         PeopleEditCommand command = (PeopleEditCommand) parser.parseCommand(PEOPLE_COMMAND_TYPE + " "
-                + PeopleEditCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased() + " "
-                + PersonUtil.getEditPersonDescriptorDetails(descriptor));
+            + PeopleEditCommand.COMMAND_WORD + " " + INDEX_FIRST_PERSON.getOneBased() + " "
```
Comment (CheyanneSim) [inline src/test/java/seedu/address/logic/parser/SharkieParserTest.java]: Syinyi, I think u need to set Intellij to indent at 8 spaces

File: src/test/java/seedu/address/model/transaction/DebtTest.java [hunk 5]
```diff
@@ -22,6 +23,14 @@ public void equals() {
                 .withDescription("Supper")
                 .withAmount("10")
                 .withDate("2020-01-23").build();
+        Loan supperLoan = new LoanBuilder()
```
Comment (CheyanneSim) [inline src/test/java/seedu/address/model/transaction/DebtTest.java:26]: hmmm i think this shouldn't be in DebtTest?

**Review by CheyanneSim — COMMENTED** [review]
I think can merge after a few changes :)

File: src/main/java/seedu/address/model/transaction/Loan.java (hunk 1 above)
Comment (syinyichen) [reply src/main/java/seedu/address/model/transaction/Loan.java]: But, do we need to differentiate them from other transactions?

File: src/test/java/seedu/address/model/transaction/DebtTest.java (hunk 5 above)
Comment (syinyichen) [reply src/test/java/seedu/address/model/transaction/DebtTest.java:26]: this is to check if they are the different type of transactions

File: src/main/resources/view/PersonListCard.fxml (hunk 3 above)
Comment (syinyichen) [reply src/main/resources/view/PersonListCard.fxml]: I think intellij automatically changed to wildcard.

**Pull request merged**